src/sonic-yang-models/yang-models/sonic-system-aaa.yang

```diff
@@ -30,8 +30,12 @@ module sonic-system-aaa {
                 }
 
                 leaf login {
-                    type string;
-                    description "AAA authentication/authorization/accounting methods - local/tacacs+/disable";
+                    type string {
+                        pattern 'radius|tacacs\+|local|default' {
```

Comment:
Do we accommodate the value "default" in SONiC auth operations?

Reply:
We do accommodate "default", the default will set authentication "login" parameter to "local" and remove all other authentication settings.